packages/middleware-io/src/json-schema.ts

```diff
@@ -0,0 +1,32 @@
+import * as t from 'io-ts';
+
+interface JSONSchemaArrayRuntimeType extends t.ArrayType<JSONSchemaRuntimeType> {
+}
+
+type JSONSchemaObjectRuntimeType = t.InterfaceType<{ [key: string]: JSONSchemaRuntimeType }>
+type JSONSchemaRuntimeType =
+  | t.StringType
+  | t.NumberType
+  | t.BooleanType
+  | JSONSchemaArrayRuntimeType
+  | JSONSchemaObjectRuntimeType
+
+export const toJSONSchema = (type: JSONSchemaRuntimeType): any => {
```

Comment:
Actually this one is a left over from one of attempts to extract data from io. I'll remove it from this PR and it will probably land in next PR as an improvement.